.gitignore

```diff
@@ -39,3 +39,5 @@ bin/
 .factorypath
 .settings
 .springBeans
+src/test/java/org/c4sg/ProjectControllerTest.java
```

Comment:
1. `src/test/java/org/c4sg/ProjectControllerTest.java` is source code, it should not be added to gitignore. Please revert the change on gitignore.
2. FYI - To prevent unit tests from causing Travis Build failure, all the unit tests are ignored by using spring ignore annotation.
![image](https://cloud.githubusercontent.com/assets/24813620/25286066/0c9f06a8-2682-11e7-81fd-953f5df79718.png)